modules/account-lib/package.json

```diff
@@ -39,7 +39,7 @@
     "@bitgo/blake2b": "^3.0.2",
     "@bitgo/bls": "^0.1.0",
     "@bitgo/bls-dkg": "^1.0.1",
-    "@bitgo/statics": "^6.16.0",
+    "@bitgo/statics": "6.16.0",
```

Comment:
Is there a reason the carat was removed here? This will get re-added when doing a release

Reply:
there's a lint step in bgms drone that requires all dependencies within the build system use the same exact version. I thought the `^` here was a typo and the exact version was preferred, but we can make the umbrella packages in bgms use the carat instead if the release is going to put it in there every time anyways.